tests/helpers.py

```diff
@@ -1907,3 +1908,27 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def skipif_ocs_version(expr):
```

Comment:
What about to move it here:  https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/testlib.py

From here we import all the markers, base class and so on so testlib sounds good to me for such functionality. WDYT?

Reply:
done